.github/workflows/rector_install.yaml

```diff
@@ -18,9 +18,7 @@ jobs:
                     - "7.3"
                     - "7.4"
                     - "8.0"
-                composer-versions:
-                    - "v1"
-                    - "v2"
+
```

Comment:
This should be fine, as composer 2 is here almost a year :)
Also our plugin is not using the composer API much.